web/src/styles/scss/login.scss

```diff
@@ -0,0 +1,14 @@
+.login-card {
+	border-left: 16px solid $color-error;
+	margin-top: 4em;
+	margin-bottom: 4em;
+	background-color: $color-white;
+	padding: 35px;
+	max-width: 42rem;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.login-card img {
+	margin-left: -11px;
+}
```

Comment:
You should configure your editor to add a new line character to the end of files. Sounds silly, but this is how POSIX defines a line: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795